pkg/reconciler/inmemorychannel/controller/inmemorychannel.go

```diff
@@ -253,6 +259,24 @@ func (r *Reconciler) reconcileChannelService(ctx context.Context, imc *v1alpha1.
 	return svc, nil
 }
 
+func (r *Reconciler) createSubscribableStatus(subscribable *eventingduck.Subscribable) *eventingduck.SubscribableStatus {
+	if subscribable == nil {
+		return nil
+	}
+	subscriberStatus := make([]eventingduck.SubscriberStatus, 0)
+	for _, sub := range subscribable.Subscribers {
+		status := eventingduck.SubscriberStatus{
+			UID:                sub.UID,
+			ObservedGeneration: sub.Generation,
+			Ready:              corev1.ConditionTrue,
```

Comment:
When looking at the v1beta1 api, questions came up on why this ready is here. What does it mean?

Reply:
It's being used by Subscription. I think the idea is to mark a subscription ready when it has been processed by the channel.